docs/usage/export.md

```diff
@@ -20,19 +20,22 @@ With the OpenCTI platform, you can manually export your intelligence content in
 To export one or more entities you have two possibilities. First you can click on the button "Open export panel". The list of pre-existing exports will open, and in the bottom right-hand corner you can configure and generate a new export.
 
 ![open export panel](assets/open_export_panel.png)
-
 This opens the export settings panel, where you can customize your export according to three fields:
 
 - desired export format (text/csv, application/pdf, application/vnd.oasis.stix+json, text/plain)
 - export type (simple or full),
-- the max marking definition level of the elements in the entity to be exported (a TLP level, for instance).
+- the max marking definition levels of the elements in the entity to be exported (a TLP level, for instance).
```

Comment:
```suggestion
- the max marking definition levels of the elements to be included in the export (a TLP level, for instance). For a marking definition type to be taken into account here, a marking definition from this type must be provided. For example, if you select TLP:GREEN for this field, AMBER and RED elements will be excluded but it will not take into account any PAP markings unless one is elected too.
```